src/lib/replication/repl_dev/raft_repl_dev.cpp

```diff
@@ -512,6 +522,9 @@ void RaftReplDev::fetch_data_from_remote(std::vector< repl_req_ptr_t > rreqs) {
             auto raw_data = e.value().response_blob().cbytes();
             auto total_size = e.value().response_blob().size();
 
+            COUNTER_INCREMENT(m_metrics, fetch_rreq_cnt, 1);
```

Comment:
Aren't we double incrementing the fetch_rreq_cnt, one while requesting and other after response?